profiles/features/musl/package.mask

```diff
@@ -475,3 +475,9 @@ sys-block/wait-for-dri-devices-rules
 
 # rust-bin isn't available on all platforms
 dev-lang/rust-bin
+
+# Cheyenne Wills <cwills@witznd.net> (2025-01-05)
```

Comment:
New masks go at the top.

Reply:
I'll wait a bit to see if there are any other comments, then I'll update the commit.

Reply:
I'm happy to merge it now once this is sorted and also you've adopted it ideally: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Taking_over_an_existing_package and https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml.

Reply:
(just copy your entry from icon)